src/main/scala/scalaswingcontrib/test/TreeDemo.scala

```diff
@@ -151,7 +151,9 @@ object TreeDemo extends SimpleSwingApplication {
     val insertBeforeButton = new Button(Action("Insert before") {
       val pathToInsertBefore = pretendFileTree.selection.paths.leadSelection
       for (path <- pathToInsertBefore) {
-        val succeeded = pretendFileTree.model.insertBefore(path, PretendFile("new-before-" + path.last.name))
+        val succeeded = if (path.lengthCompare(1)>0) {
```

Comment:
I am used to do it this way on `Seq`s, to to degree I consider it idiomatic, because `length` can be O(N) (e.g. on List), but in this can `path.length > 1` is good, as Path is `IndexedSeq`, therefore even this is O(1). If you prefer it, I will change it that way.